optimade/server/logger.py

```diff
@@ -37,23 +42,25 @@
 
 try:
     LOGS_DIR.mkdir(exist_ok=True)
+
+    # Handlers
+    FILE_HANDLER = logging.handlers.RotatingFileHandler(
+        LOGS_DIR.joinpath("optimade.log"), maxBytes=1000000, backupCount=5
+    )
```

Comment:
Is there a particular reason why you shifted this up here?

Reply:
My first attempt to test it set `log_dir` to `/var/log`, so no permission error was triggered for the directory (as it existed), but the `RotatingFileHandler` raised a `PermissionError` instead. I guess it could be moved into its own try/except to give a better error message?

Reply:
Ah great! Then I applaud the change. I think the warning message already explains this might happen as well, right?

Reply:
Yup, it just wasn't catching it before.